charts/newrelic-infrastructure/values.yaml

```diff
@@ -12,23 +12,20 @@ images:
     tag: 1.23.0
     # -- Image for the agent sidecar.
     repository: newrelic/k8s-events-forwarder
-    registry: docker.io
```

Comment:
shouldn't we keep `registry:` to show the user they can change that?

Reply:
I think what @kang-makes had in mind is to support advanced use cases where users can configure their CRI so unqualified images are redirected to registries that are not `docker.io`. But I think we should leave the entry:
```yaml
registry: 
```

Reply:
@roobre hit the point: https://github.com/containers/image/blob/main/docs/containers-registries.conf.5.md#normalization-of-dockerio-references

I don't like that `docker.io` is the de-facto standard but it is that way. You can set in the CRI what registry you want to point to. So the sysadmin could be able to point an `unqualified search registries` to wherever they want.

Solved in [#249f236](https://github.com/newrelic/nri-kubernetes/pull/383/commits/249f236fc5b7dcbed0dd346ff38793e28f14be3f)